.github/workflows/ci.yml

```diff
@@ -10,14 +10,38 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-    
-      - name: Set up Python 
+
+      - name: Set up Python
         uses: actions/setup-python@v4
         with:
           python-version: "3.x"
-      
-      - name: Install pypa/build
-        run: python3 -m pip install build poetry --user
+      - name: Install Poetry
+        uses: snok/install-poetry@v1
```

Comment:
Why unnecessary introduce another dependency to such a small project?
We can just do the install of poetry as before.

```yaml
      - name: Install pypa/build
        run: python3 -m pip install build poetry --user
```